src/utils.ts

```diff
@@ -1,3 +1,22 @@
+import { readdir } from 'fs/promises'
+import { resolve } from 'path'
+
+export const isNuxtAppDir = async (dir: string) => {
```

Comment:
Since function is not returning anything but ensuring, I think would be better calling it `ensureNuxtAppDir`
```suggestion
export async function ensureNuxtAppDir (dir: string) {
```

Reply:
And perhaps to make it shorter – `ensureNuxtApp(dir: string)`. Clear enough?